docs/pages/careers.tsx

```diff
@@ -144,18 +144,18 @@ const faqData = [
 ];
 
 const openRolesData = [
-  // {
-  //   // 1
-  //   title: 'Product',
-  //   roles: [
-  //     {
-  //       title: 'Product Designer',
-  //       description:
-  //         'Design is critical to the success of our mission. We are looking for skills that complement our Lead Designer. You will empower our audience that seeks to build outstanding-looking UIs with new tools.',
-  //       url: '/company/11-product-designer/',
-  //     },
-  //   ],
-  // },
+  {
+    // 1
+    title: 'Product',
+    roles: [
+      {
+        title: 'Designer',
+        description:
+          'Design is critical to the success of our mission. We are looking for skills that complement our Lead Designer. You will empower our audience that seeks to build outstanding-looking UIs with new tools.',
+        url: '/company/11-product-designer/',
```

Comment:
URL to match the role name?

```suggestion
        url: '/company/11-designer/',
```